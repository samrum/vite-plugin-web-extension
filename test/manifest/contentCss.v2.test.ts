import { getResourceDir, runManifestV2Test } from "./manifestTestUtils";

const resourceDir = getResourceDir("contentCss");

runManifestV2Test("contentCss", () => ({
  content_scripts: [
    {
      js: [`${resourceDir}/content.js`],
      css: [`${resourceDir}/content.css`],
      matches: ["https://*/*", "http://*/*"],
    },
  ],
}));
