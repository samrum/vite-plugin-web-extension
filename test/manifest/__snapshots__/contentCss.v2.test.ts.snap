// Vitest Snapshot v1

exports[`contentCss - Manifest V2 1`] = `
[
  {
    "code": "console.log(\\"content\\");
",
    "dynamicImports": [],
    "exports": [],
    "facadeModuleId": "vite-plugin-web-extension/test/manifest/resources/contentCss/content.js",
    "fileName": "assets/test/manifest/resources/contentCss/content.js",
    "implicitlyLoadedBefore": [],
    "importedBindings": {},
    "imports": [],
    "isDynamicEntry": false,
    "isEntry": true,
    "isImplicitEntry": false,
    "map": null,
    "modules": {
      "vite-plugin-web-extension/test/manifest/resources/contentCss/content.js": {
        "code": "console.log(\\"content\\");",
        "originalLength": 24,
        "removedExports": [],
        "renderedExports": [],
        "renderedLength": 23,
      },
    },
    "name": "test/manifest/resources/contentCss/content",
    "referencedFiles": [],
    "type": "chunk",
    "viteMetadata": {
      "importedAssets": Set {},
      "importedCss": Set {},
    },
  },
  {
    "fileName": "test/manifest/resources/contentCss/content.css",
    "name": undefined,
    "source": ".css {
}
",
    "type": "asset",
  },
  {
    "fileName": "manifest.json",
    "name": undefined,
    "source": "{
  \\"version\\": \\"1.0.0\\",
  \\"name\\": \\"Manifest Name\\",
  \\"manifest_version\\": 2,
  \\"content_scripts\\": [
    {
      \\"js\\": [
        \\"assets/test/manifest/resources/contentCss/content.js\\"
      ],
      \\"css\\": [
        \\"test/manifest/resources/contentCss/content.css\\"
      ],
      \\"matches\\": [
        \\"https://*/*\\",
        \\"http://*/*\\"
      ]
    }
  ]
}",
    "type": "asset",
  },
]
`;
